Say/Say.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  Say.swift
+//  Say
+//
+//  Created by Seungwon Kim on 2016. 9. 24..
+//  Copyright © 2016년 youknowone.org. All rights reserved.
+//
+
+import Cocoa
+
+/**
+    Easy-to-use interface for `say` command in OS X.
+*/
+class SayAPI: NSObject {
+    let speechSynthesizer: NSSpeechSynthesizer?
```

Comment:
Because the synthesizer is initialized in initializer on line 34, the type doesn't need to be optional. I recommend to use `NSSpeechSynthsizer` not `NSSpeechSynthesizer?`